.github/workflows/test.yml

```diff
@@ -37,6 +39,12 @@ jobs:
       # Use always() to always run this step to publish test results when there are test failures
       if: ${{ always() }}
```

Comment:
Can this be removed, now that we are using `Coverage.py` for the coverage?

Reply:
We still need the junit test results to show errors with the unit tests beside coverage.